doc/code/qml_fermi.rst

```diff
@@ -0,0 +1,89 @@
+qml.fermi
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating fermionic operators.
+
+.. currentmodule:: pennylane.fermi
+
+.. automodapi:: pennylane.fermi
+    :no-heading:
+    :no-main-docstr:
+    :no-inherited-members:
+
+FermiC and FermiA
+-----------------
+
+The fermionic creation and annihilation operators are constructed with :class:`~pennylane.FermiC`
+and :class:`~pennylane.FermiA`, respectively, by passing the index of the orbital that the fermionic
+operator acts on. For instance, the operators :math:`a^{\dagger}_0` :math:`a_3` are constructed as
+
+>>> qml.FermiC(0)
+a⁺(0)
+>>> qml.FermiA(3)
+a(3)
+
+These operators can be multiplied by each other to create :math:`n`-orbital fermionic operators such
+as :math:`a^{\dagger}_0 a_0 a^{\dagger}_3 a_3` that we call a Fermi word.
+
+>>> qml.FermiC(0) * qml.FermiA(0) * qml.FermiC(3) * qml.FermiA(3)
+a⁺(0) a(0) a⁺(3) a(3)
+
+The Fermi words can be linearly combined to create fermionic operators that we call a Fermi
+Sentence. For instance, a fermionic Hamiltonian such as
```

Comment:
```suggestion
sentence. For instance, a fermionic Hamiltonian such as
```
Looks like we're going with lowercase everywhere else, but this one is still capitalized